portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +26,33 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String[] wordArray = {"Hey!", "Check", "here!"};
+    /**
     String toJson = convertToJson(wordArray);
-
     response.setContentType("application/json;");
     response.getWriter().println(toJson);
+    */
   }
-  private String convertToJson(String[] wordArray) {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String getMessage = getParameter(request, "comment", "");
+    String submissionResponse = "Submitted! Thank you " + getParameter(request, "name", "");
+
+    response.setContentType("text/html;");
+    response.getWriter().println("<h3>" + submissionResponse + "</h3>");
+    response.getWriter().println("<p> <b>message sent:</b> <i>" + getMessage + "</i></p>");
+  }
+
+  /**private String convertToJson(String[] something) {
```

Comment:
Same here. It's almost fair to say in a repository with many developers contributed together, a piece of commented code should be deleted unless it serves very special purpose. For example, it's an example for you to illustrate a problem etc., which doesn't seems to be the case here.